client-core/src/main/java/com/influxdb/exceptions/InfluxException.java

```diff
@@ -118,22 +119,23 @@ public int status() {
      * @return a response body
      */
     @Nonnull
-    public JSONObject errorBody() {
+    public JsonObject errorBody() {
```

Comment:
This is a breaking change for our public API. Will we change a major version of client?

What about use a generic map `Map<String, Object>` to avoid dependency to third party library?

Reply:
Yes, we should increase major version. Completely remove json from public api is better.